builds/build.go

```diff
@@ -0,0 +1,93 @@
+package build
+
+import (
+  "fmt"
+  "time"
+
+  "github.com/Shyp/go-circle"
+  "github.com/Shyp/go-git"
+)
+
+func inSlice(a string, list []string) bool {
+  for _, b := range list {
+    if b == a {
+        return true
+    }
+  }
+  return false
+}
+
+// GetBuilds gets the status of the 5 most recent Circle builds for a branch
+func GetBuilds(branch string) error {
+  // Different statuses Circle builds can have
+  green := []string{"fixed", "success"}
+  grey  := []string{"retried", "canceled", "not_run"}
+  red   := []string{"infrastructure_fail", "timedout", "failed", "no_tests"}
+  blue  := []string{"running"}
+  // purple := []string{"queued", "not_running", "scheduled"}
+
+  tip, err := git.Tip(branch)
+  _ = tip
+  // This throws if the branch doesn't exist
+  if err != nil {
+    return err
+  }
+
+  fmt.Println("\nFetching recent builds for", branch, "starting with most recent commit\n")
+  // Give CircleCI a little bit of time to start
+  time.Sleep(1 * time.Second)
+
+  remote, err := git.GetRemoteURL("origin")
+  if err != nil {
+    return err
+  }
+
+  cr, err := circle.GetTree(remote.Path, remote.RepoName, branch)
+  if err != nil {
+    return err
+  }
+
+  sum := 0
+
+  // Limited to 5 most recent builds. Feature would be to pass in number
+  // of builds to fetch via command line args
+  for i := 0; i < 5; i++ {
+    build := (*cr)[i]
+    ghUrl, url, status := build.CompareURL, build.BuildURL, build.Status
+
+    // Based on the status of the build, change the color of status print out
+    if inSlice(status, green) {
```

Comment:
Oh! It's not super obvious, but we should be able to get to this by calling `build.Passed()`, which checks this:

```go
func (tb TreeBuild) Passed() bool {
	return tb.Status == "success" || tb.Status == "fixed"
}
```

More here: https://godoc.org/github.com/Shyp/go-circle#TreeBuild.Passed